Makefile

```
@@ -162,6 +162,10 @@ start-publication-watchdog: ## Start the publication watchdog
 help: ## Show this help
 	@echo "Use 'make [command]' to run one of these commands:"
 	@echo ""
-	@fgrep --no-filename "##" ${MAKEFILE_LIST} | head -n '-1' | sed 's/\:.*\#/\: \#/g' | column -s ':#' -t -c 2
+ifeq ($(shell uname), Darwin)
+	@fgrep --no-filename "##" ${MAKEFILE_LIST} | tail -r | tail -n +3 | tail -r | sed 's/\:.*\#/\: \#/g' | column -s ':#' -t -c 2 | sed 's/(null)//g'
+else
+	@fgrep --no-filename "##" ${MAKEFILE_LIST} | head -n '-2' | sed 's/\:.*\#/\: \#/g' | column -s ':#' -t -c 2  # assume GNU tools
```

Je pense que garder la commande telle qu'elle est actuellement pour Linux devrait corriger la petite différence d'affichage ! 
```suggestion
	@fgrep --no-filename "##" ${MAKEFILE_LIST} | head -n '-1' | sed 's/\:.*\#/\: \#/g' | column -s ':#' -t -c 2  # assume GNU tools
```

---

Non : comme mon patch rajoute des lignes dans le Makefile qu'on ne veut pas afficher dans l'aide, il faut adapter le nombre de ligne qu'on enlève. Si je passe la valeur à -1, j'obtiens l'affichage suivant sur Linux : 
```
....
help                            Show this help
	@fgrep --no-filename "         " ${MAKEFILE_LIST} | tail -r | tail -n +3 | tail -r | sed 's/\                                                             ' -t -c 2 | sed 's/(null)//g'
```